lib/appmap/gem_hooks/activerecord.yml

```diff
@@ -1,2 +1,4 @@
 - method: ActiveRecord::Relation#records
   label: dao.materialize
+- method: ActiveRecord::FixtureSet::File#raw_rows
+#  label: deserialize.safe
```

Comment:
Why are these labels commented out?

Reply:
And why would they be needed anyway?

Reply:
An example would be `YAML.load` - while this method is unsafe, it could be used in a context that guarantees it's actually safe. For example, Rails loads configuration with `YAML.load`. Personally, I wonder why they don't use `safe_load`. But, since the Rails configuration is subject to developer approval, a good argument can be made that the data itself is safe, and therefore even though `YAML.load` is unsafe, using YAML.load to load Rails configuration is not actually unsafe, and therefore we shouldn't report a finding for such a situation.

In general, an event labeled `deserialize.unsafe` that is a descendant of an event labeled `deserialize.safe` will not trigger a finding.

Reply:
Thanks, that makes a lot of sense.